code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm

```diff
@@ -84,6 +84,14 @@
 
 /mob/living/simple_animal/hostile/abnormality/despair_knight/proc/BlessedDeath(datum/source, gibbed)
 	SIGNAL_HANDLER
+	blessed_human.cut_overlay(mutable_appearance('ModularTegustation/Teguicons/tegu_effects.dmi', "despair", -MUTATIONS_LAYER))
+	UnregisterSignal(blessed_human, COMSIG_LIVING_DEATH)
+	UnregisterSignal(blessed_human, COMSIG_HUMAN_INSANE)
+	blessed_human.physiology.red_mod /= 0.5
+	blessed_human.physiology.white_mod /= 0.5
+	blessed_human.physiology.black_mod /= 0.5
+	blessed_human.physiology.pale_mod /= 2
```

Comment:
Dividing or multiplying will inevitably cause issues with other things that change damage modifiers, so we need this changed(both here and in the proc that gives blessing).